pkgs/top-level/all-packages.nix

```diff
@@ -36275,6 +36275,7 @@ with pkgs;
   root = callPackage ../applications/science/misc/root {
     python = python3;
     inherit (darwin.apple_sdk.frameworks) Cocoa CoreSymbolication OpenGL;
+    stdenv = if stdenv.isLinux then gcc11Stdenv else stdenv;
```

Comment:
Is this specific to aarch64-linux?

Reply:
Yes. On x86_64-linux stdenv defaults to gcc11Stdenv, so there is no change.

Reply:
Okay, but what happens if x86 stdenv switches to gcc12, but aarch64 doesn't?

Reply:
Also if you could leave a comment on what error this fixes?

Reply:
> Okay, but what happens if x86 stdenv switches to gcc12, but aarch64 doesn't?

Nice argument. Switched to aarch64-linux only and added comment.